package.json

```diff
@@ -7,7 +7,7 @@
     "url": "git://github.com/ulfryk/angular-typescript.git"
   },
   "scripts": {
-    "postinstall": "bower install",
+    "postinstall": "bower install && typings install",
```

Comment:
I was confused by this at first, as I thought it was a hook that ran automatically with `npm`. Now I see that it is a run script. So in `CONTRIBUTING.md`, doesn't both `bower install` _and_ `typings install` now need to exist?